sample01.go

```diff
@@ -1,7 +1,15 @@
 package sample01
 
+import "fmt"
+
 func HelloWorld(s string) string {
-	var x int
-	x = "string"
+	var y int
+	y = "aaa" // TODO
 	return "hello world, " + s
 }
+
+var ExportedVarWithoutComment = 1 // TODO: revert
+
+func Greet() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function Greet should have comment or be unexported